SPECS/etcd/etcd-3.4.13.spec

```diff
@@ -0,0 +1,121 @@
+Summary:        A highly-available key value store for shared configuration
+Name:           etcd
+Version:        3.4.13
+Release:        1%{?dist}
+License:        ASL 2.0
+URL:            https://github.com/etcd-io/etcd/
+Group:          System Environment/Security
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+#Source0:       https://github.com/etcd-io/%{name}/archive/v%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+Source1:        etcd.service
+
+BuildRequires:  golang >= 1.13
+BuildRequires:  git
+
+%description
+A highly-available key value store for shared configuration and service discovery.
+
+%prep
+%setup -q
+
+%build
+# Turn off auto moduling. golang 1.13 does not automatically consider the vendor folder (it does as of 1.14).
```

Comment:
Why not simply depend on golang >=1.14 then? If not, shouldn't we guard this bit to do different things for 1.14+?

Reply:
technically etcd requires golang 1.13 and I think we'd better stick with what etcd stated (1.13+)
fyi, I just kept the existing etcd spec as it was